assets/js/search.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * To perform client side searching. 
+ */
+(function () {
+    $('#search-box').keyup(function(){
+        var searchQuery = $(this).val();
+        
+        if(searchQuery.length === 0)  {
+            $('#filtered-records').html('');
+            $('#listings').show();
+            return;
+        } else {
+            // Hide the initial data before search began
+            $('#listings').hide();
+
+            var regex = new RegExp(searchQuery, "i");
+            var output = '<div class="row">';
```

Comment:
Assembling HTML like this is error prone, hard to maintain, and makes injection attacks possible. 

What about adding a CSS class to elements that don't match the search query to hide them? So the page stays largely the same, but certain items hide / show? 

If you're not sure what the search UI should look like, feel free to ping Sam!

Reply:
Writing it was also hard :) meaning it will be hard to maintain. I wanted someone to give feedback.
I made use of jquery .hide() and .show() to implement this so the page remains the same